src/client/vendor/steemitHtmlReady.js

```diff
@@ -122,9 +122,8 @@ function link(state, child) {
     state.links.add(url);
     if (state.mutate) {
       // If this link is not relative, http, or https -- add https.
-      if (!/^[\w.-]+:(\/\/)?/.test(url)) {
-        child.setAttribute('href', `https://${url}`);
-      }
+      if (/^\//.test(url)) child.setAttribute('href', `https://busy.org${url}`);
```

Comment:
We shouldn't use hardcoded `https://busy.org` URL. It won't work on localhost, preview apps, beta version and forks.

Reply:
Thanks for your comment. I've removed it now.